user-manager-service/src/main/resources/db/changelog/changelog-master.xml

```diff
@@ -3,6 +3,6 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 
-    <include file="user-manager_init_schemas.xml" relativeToChangelogFile="true"/>
+    <include file="create-changeset-users-table.xml" relativeToChangelogFile="true"/>
```

Comment:
it would be nice to add numbers (months) to changelog files
2023.11_initial_scripts.xml

BTW it would be nice to decide and provide the same naming rules for all services

Reply:
By dates you mean date of changelog creation or changelog application? And cause our app in initial  development phase I don't see reasons for that

Reply:
new version need to be checked